nixos/modules/services/web-apps/snips-sh.nix

```diff
@@ -0,0 +1,156 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib)
+    mkOption
+    mkEnableOption
+    mkPackageOption
+    mapAttrs
+    optional
+    boolToString
+    isBool
+    mkIf
+    getExe
+    types
+    ;
+
+  cfg = config.services.snips-sh;
+in
+{
+  meta.maintainers = with lib.maintainers; [
+    isabelroses
+    NotAShelf
+  ];
+
+  options.services.snips-sh = {
+    enable = mkEnableOption "snips.sh";
+
+    package = mkPackageOption pkgs "snips-sh" {
+      example = "pkgs.snips-sh.override {withTensorflow = true;}";
+    };
+
+    stateDir = mkOption {
+      type = types.path;
+      default = "/var/lib/snips-sh";
+      description = "The state directory of the service";
```

Comment:
Other option descriptions have a period at the one, perhaps this one should have one as well.